pysgrid/tests/test_interpolation.py

```diff
@@ -0,0 +1,212 @@
+    res = (translated_indices == answer).all()
+    assert(res)
+
+    # Test case for places where edge1 increases slope counterclockwise
+    te1_node_lon = np.array(([2, 4, 6], [2, 4, 6], [1, 3, 5]))
+    te1_node_lat = np.array(([1, 1, 1], [3, 3, 3], [5, 5, 5]))
+    te1_edge1_lon = np.array(
+        ([2, 4, 6], [2, 4, 6], [1.5, 3.5, 5.5], [0.5, 2.5, 4.5]))
+    te1_edge1_lat = np.array(([0, 0, 0], [2, 2, 2], [4, 4, 4], [6, 6, 6]))
+    test_points = np.array(([1.99, 3], [3.99, 3.01], [3.99, 2.99], [5.99, 3]))
+
+    g = SGrid(node_lon=te1_node_lon,
+              node_lat=te1_node_lat,
+              edge1_lon=te1_edge1_lon,
+              edge1_lat=te1_edge1_lat)
+
+    indices = g.locate_faces(test_points)
+    translated_indices = g.translate_index(test_points, indices, 'edge1')
+    answer = np.ma.masked_array(([-1, -1], [1, 1], [1, 1], [-1, -1]),
+                                mask=[[True, True], [False, False], [False, False], [True, True]])
+    res = (translated_indices == answer).all()
+    assert(res)
+
+    # Test case for places where edge2 increases slope counterclockwise
+    te2_node_lon = np.array(([1, 3, 5], [1, 3, 5], [1, 3, 5]))
+    te2_node_lat = np.array(([1, 1, 2], [3, 3, 4], [5, 5, 6]))
+    te2_edge2_lon = np.array(
+        ([0, 2, 4, 6], [0, 2, 4, 6], [0, 2, 4, 6]))
+    te2_edge2_lat = np.array(
+        ([1, 1, 1.5, 2.5], [3, 3, 3.5, 4.5], [5, 5, 5.5, 6.5]))
+    test_points = np.array(([3, 1.01], [3.01, 3.01], [2.99, 3.01], [3, 5.01]))
+
+    g = SGrid(node_lon=te2_node_lon,
+              node_lat=te2_node_lat,
+              edge2_lon=te2_edge2_lon,
+              edge2_lat=te2_edge2_lat)
+
+    indices = g.locate_faces(test_points)
+    translated_indices = g.translate_index(test_points, indices, 'edge2')
+    answer = np.ma.masked_array(([-1, -1], [0, 1], [0, 1], [-1, -1]),
+                                mask=[[True, True], [False, False], [False, False], [True, True]])
+    res = (translated_indices == answer).all()
+    assert(res)
+
+    # Test case for places where edge2 increases slope clockwise
+    te2_node_lon = np.array(([1, 3, 5], [1, 3, 5], [1, 3, 5]))
+    te2_node_lat = np.array(([2, 2, 1], [4, 4, 3], [6, 6, 5]))
+    te2_edge2_lon = np.array(
+        ([0, 2, 4, 6], [0, 2, 4, 6], [0, 2, 4, 6]))
+    te2_edge2_lat = np.array(
+        ([2, 2, 1.5, 0.5], [4, 4, 3.5, 2.5], [6, 6, 5.5, 4.5]))
+    test_points = np.array(([3, 1.99], [3.01, 3.99], [2.99, 3.99], [3, 5.99]))
+
+    g = SGrid(node_lon=te2_node_lon,
+              node_lat=te2_node_lat,
+              edge2_lon=te2_edge2_lon,
+              edge2_lat=te2_edge2_lat)
+
+    indices = g.locate_faces(test_points)
+    translated_indices = g.translate_index(test_points, indices, 'edge2')
+    answer = np.ma.masked_array(([-1, -1], [1, 1], [1, 1], [-1, -1]),
+                                mask=[[True, True], [False, False], [False, False], [True, True]])
+    res = (translated_indices == answer).all()
+    assert(res)
```

Comment:
How did you come to that conclusion? `res` will be True only if all the answers match, and False every time else.